src/inverseRobot.js

```diff
@@ -24,7 +24,28 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const mainPropArray = [];
+  const robotReverse = {};
+
+  for (const key in robot) {
+    mainPropArray.push(key, robot[key]);
+  }
+
+  mainPropArray.sort();
+
+  for (let i = 0; i < mainPropArray.length; i++) {
+    if (mainPropArray[i] === mainPropArray[i + 1]) {
+      return null;
+    }
+  }
+
+  Object.keys(robot).forEach(function(value) {
+    const key = robot[value];
```

Comment:
It's better to get used to arrow function from the start.

```suggestion
  Object.keys(robot).forEach((value) => {
    const key = robot[value];
```